pkg/grizzly/jsonnet.go

```diff
@@ -28,9 +28,16 @@ func newFileLoader(fi *jsonnet.FileImporter) importLoader {
 	}
 }
 
-func newExtendedImporter(path string, jpath []string) *ExtendedImporter {
-	absolutePaths := make([]string, len(jpath)+1)
+func newExtendedImporter(jsonnetFile, path string, jpath []string) *ExtendedImporter {
+	absolutePaths := make([]string, len(jpath)*2+1)
 	absolutePaths = append(absolutePaths, path)
+	jsonnetDir := filepath.Dir(jsonnetFile)
+	for _, p := range jpath {
```

Comment:
This for loop can be merged with the next one, by setting a `temp := p` var and then append both?

Reply:
I did think of that at first. But I want to have the two dirs relative to the jsonnet file first, then the two relative to the current directory.